input/pagecontent/reference.md

```diff
@@ -979,7 +986,6 @@ Rules types that apply to Instances are: [Assignment](#assignment-rules), [Inser
   ```
   Instance:   DrDavidAnydoc
   InstanceOf: http://hl7.org/fhir/us/core/StructureDefinition/us-core-practitioner
-  Title:  "Dr. David Anydoc"
```

Comment:
`Title` keyword is useless in an inline instance since it's only used when populating the IG JSON w/ references to examples (and inline instances are not examples and do not get referenced in the IG JSON).

Also, it felt like this example was redundant w/ the next one -- and the next one acts as if it is _introducing_ ideas that this (previous) example already used.  So you might want to just delete this whole example.